### DISCUSSION TITLE
MlFlow Implementation

### DISCUSSION THREAD

**Discussion by fabiogeraci** [pytorch/torchtune#2103] [Ideas]
I developed an initial Class to implement MlFlow, would the community be interested?

**Comment by steveepreston** [comment]
> I developed an initial Class to implement MlFlow, would the community be interested?

100% Please
The lack of `mlflow` integration in this library is so painful.

**Reply by fabiogeraci** [reply]
I am on AL, once back i Will try to work on the PR.

**Reply by steveepreston** [reply]
@fabiogeraci
Thanks Man.
Have you any temporary alter way for now?

**Reply by fabiogeraci** [reply]
Do not take work in AL

**Reply by steveepreston** [reply]
@fabiogeraci 😂

**Comment by ebsmothers** [comment]
@fabiogeraci given there is some interest from the community happy to review a PR (once you're off leave)

**Comment by nathan-az** [comment]
I require this anyway and was planning to work off a fork, but since there's interest I've raised a PR https://github.com/pytorch/torchtune/pull/2365

Happy for this to merge if the team is happy, or for it to work as a reference for @fabiogeraci.

**Comment by pbontrager** [comment]
It's landed now! Thanks everyone who contributed!

**Reply by steveepreston** [reply]
Nice!
